dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -144,3 +144,5 @@ authentication-ip.Student = 6.6.6.6
 useProxies = true
 proxies.trusted.ipranges = 7.7.7.7
 proxies.trusted.include_ui_ip = true
+
+management.health.solrOai.enabled = false
```

Comment:
Could we add a comment as to why this is needed?  It looked odd to me at first, but it looks like you are disabling the OAI health check because it will always fail in the test environment?

It's just good to add notes to other developers as to why some setting needs to be made in the "global" local.cfg for all tests.

Reply:
I had to disable this health indicator because there isn't a mock server for it and so the build was working locally but not here on github.